src/TED/TED.Utils/Tokenizer.cs

```diff
@@ -38,6 +41,27 @@ public static string ReplaceTokens(string input)
 
             return input;
         }
+        private static string GetSerial()
+        {
+            try
+            {
+                var query = new System.Management.ObjectQuery("SELECT * FROM Win32_BIOS");
+                var searcher = new System.Management.ManagementObjectSearcher(query);
+                var results = searcher.Get();
+        
+                foreach (var obj in results)
+                {
+                    return obj["SerialNumber"].ToString();
```

Comment:
This would only return the first SerialNumber found in your query - is this the intended result? Do we want to discard the remaining results?

Reply:
I've never personally run into a situation where two are output-
Is there an easy way to grab both in an array of some sort so we can handle both values?

Reply:
Replaced with `List<string> GetSerialNumbers()`